tex/chProofs.tex

```diff
@@ -1918,7 +1918,7 @@ \subsection{Proofs by induction}\label{ssec:ind}
 
 \begin{coq}{}{title=Tools}
 Lemma cats1 T s (z : T) : s ++ [:: z] = rcons s z.
-Lemma foldr_cat T f (s1 s2 : seq T) z0 :
+Lemma foldr_cat T R (f : T -> R -> R) (s1 s2 : seq T) z0 :
```

Comment:
(Note that I'm a Windows peasant right now, so I'm using https://x80.org/collacoq/ as my IDE. Maybe that's causing these problems.)